automata/regex/parser.py

```diff
@@ -181,40 +183,65 @@ def concatenate(self, other):
 
         self._final_states = other._final_states
 
-    def kleene_star(self):
+    def repeat(self, lower_bound, upper_bound):
```

Comment:
@eliotwrobson So I understand the introduction of the `repeat` method, but don't we still need to keep the `kleene_star` method lest we break API backwards-compatibility?